src/renderer/pages/LockScreen.tsx

```diff
@@ -16,20 +17,27 @@ export function LockScreen(): React.ReactElement {
 
   return (
     <Stack
+      style={{
+        backgroundImage: `url(${BackgroundImage})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+      }}
       height="100vh"
       alignItems="center"
       justifyContent="center"
       // TODO: Must be replaced with a proper inactive bacground color
       bgcolor="InactiveCaptionText"
     >
       <Stack
-        paddingX={10}
-        paddingY={6}
+        paddingX={15}
+        paddingY={8}
         gap={4}
-        bgcolor="Background"
-        border="ActiveBorder"
-        borderRadius={2}
-        boxShadow={1}
+        bgcolor="rgba(255, 255, 255, 0.8)" 
+        borderRadius={3}
```

Comment:
Don't change the border radius. It should be double the radius of the internal components. The button and text field use 1 as the border radius by default. Hence, the surrounding stack is using 2.